src/routes/admin/[classId]/module/[moduleId]/+page.svelte

```
@@ -362,26 +411,44 @@
+										)
+											? 'bg-success/5'
+											: ''}"
 										animate:flip={{ duration: 300 }}
 									>
 										<div class="flex items-center gap-3">
-											<GripVertical size={16} class="text-base-content/30 flex-shrink-0" />
+											<GripVertical size={16} class="text-base-content/30 shrink-0" />
 											<span class="text-xs font-medium text-base-content/50 w-6">#{index + 1}</span>
-											<p class="flex-1 text-sm text-base-content truncate tiptap-content-inline">{@html questionItem.stem}</p>
+											<p class="flex-1 text-sm text-base-content truncate tiptap-content-inline">
+												{@html questionItem.stem}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)